src/Mod/TechDraw/Gui/CommandCreateDims.cpp

```diff
@@ -602,6 +602,7 @@ CmdTechDrawHorizontalDimension::CmdTechDrawHorizontalDimension()
 void CmdTechDrawHorizontalDimension::activated(int iMsg)
 {
     Q_UNUSED(iMsg);
+    QMessageBox::warning(Gui::getMainWindow(),QObject::tr("Gestartet"),QObject::tr("Horizontal dimension"));
```

Comment:
What is 'Gestartet' ?

Reply:
Sorry, this statement should be commented. It's just needed during development. 'Gestartet' means 'started'